frontend/packages/console-dynamic-plugin-sdk/src/app/components/status/Status.tsx

```diff
@@ -8,13 +8,12 @@ import {
   SyncAltIcon,
   UnknownIcon,
 } from '@patternfly/react-icons';
-import { StatusComponentProps } from '../../../extensions/console-types';
 import { DASH } from '../../constants';
 import { YellowExclamationTriangleIcon } from './icons';
 import { ErrorStatus, InfoStatus, ProgressStatus, SuccessStatus } from './statuses';
-import StatusIconAndText from './StatusIconAndText';
+import { StatusIconAndText, StatusIconAndTextProps } from './StatusIconAndText';
 
-export type StatusProps = StatusComponentProps & {
+export type StatusProps = StatusIconAndTextProps & {
```

Comment:
all paths of this component end up returning a `StatusIconAndText` so this is fine to do